src/DynamicNLPModels.jl

```diff
@@ -369,16 +369,10 @@ Attributes
  - `nc` : number of algebraic constraints of the form gl <= Es + Fu <= gu
  - `nsc`: number of bounded state variables
  - `nuc`: number of bounded input variables (if `K` is defined)
- - `scaled_Jac`: placeholder to avoid allocaiton when calculating `Sigma J`
- - `J1Bx` : vector for storing multiplications when doing `Jx`
- - `J2Bx` : vector for storing multiplications when doing `Jx`
- - `J3Bx` : vector for storing multiplications when doing `Jx`
- - `J1BTx`: vector for storing multiplications when doing `J^T x`
- - `J2BTx`: vector for storing multiplications when doing `J^T x`
- - `J3BTx`: vector for storing multiplications when doing `J^T x`
- - `J1B`  : matrix for storing multiplicaitons when doing `J^T Sigma J`
- - `J2B`  : matrix for storing multiplicaitons when doing `J^T Sigma J`
- - `J3B`  : matrix for storing multiplicaitons when doing `J^T Sigma J`
+ - `SJ1`: placeholder for storing data when calculating `ΣJ`
+ - `SJ2`: placeholder for storing data when calculating `ΣJ`
+ - `SJ3`: placeholder for storing data when calculating `ΣJ`
+
 """
 struct LQJacobianOperator{T, V, M} <: LinearOperators.AbstractLinearOperator{T}
     Jac::M        # column of Jacobian block matrix
```

Comment:
Now I find this naming a bit confusing; `Jac` sounds like it's full Jacobian. Can we rename this thing? I suggest `ctrbm` https://en.wikipedia.org/wiki/Controllability but open for any other suggestion

Reply:
@sshin23 @frapac  What do you think about `truncated_Jac` or `implicit_Jac`. My initial thought is that this matrix is for simplifying storage rather than changing controlibility, since a portion of the Jacobian matrix contains variable constraints. `ctrbm` seems to me like it refers to the `block_B` matrix rather than the Jacobian, but I'm not positive.

Reply:
`truncated_jac` sounds right to me